aion/web/channels/timeout.ex

```diff
@@ -0,0 +1,42 @@
+defmodule Aion.Timeout do
+  @moduledoc """
+    This module takes care of importing timeouts from configs
+    and providing a convenient API for other modules
+    (making it possible to ask for timeouts in specific units).
+  """
+  defp from_seconds(value, unit) do
+    case unit do
+      :second -> value
+      :millisecond -> value * 1_000
+      :microsecond -> value * 1_000_000
+    end
+  end
+
+  defp get_env(key) do
+    :aion
+    |> Application.get_env(__MODULE__)
+    |> Keyword.get(key)
+  end
+
+  defp get_and_convert(key, unit) do
+    key
+    |> get_env
+    |> from_seconds(unit)
+  end
+
+  @doc """
+  Returns the timeout after which the question should be changed
+  """
+  def question_timeout(unit \\ :second), do: get_and_convert(:question_timeout, unit)
```

Comment:
I'm not sure which unit should be the default one. In code we are using mostly `:microsecond`. I could change the default unit to be so and thus DRY out the code, but getting rid of explicit unit parameters in code could probably make it less legible. I guarantee in a month we would be like: 
```
Okay now, what units am I actually using in send_after?
``` 
Seeing explicit parameters being passed in other functions' invocations would help to understand what's being done. What do you think about it?